scripts/install/install_sdk_arm.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
```

Comment:
Could we have one script for both arm / x86 and simply autodetect the required Go version (using `uname` or similar)?

Reply:
OK sure. How should I test the ARM script - on an RPI or similar? Did you test it?

Reply:
yup, tested on two RPi, now running a gaia chain

Reply:
https://github.com/tendermint/coding/issues/72